library/alt

```diff
@@ -1,7 +1,7 @@
 Maintainers: ALT Linux Team Cloud <alt-cloud@altlinux.org> (@alt-cloud),
              Alexey Shabalin <shaba@altlinux.org> (@shaba),
              Gleb Fotengauer-Malinovskiy <glebfm@altlinux.org> (@glebfm),
-             Mikhail Gordeev <obirvalger@altlunux.org> (@obirvalger)
+             Nadezhda Fedorova <fedor@altlunux.org> (@fedor)
```

Comment:
Welcome!  I think you meant @FNDenisovna here (this is a GitHub handle), right?

Reply:
Thank you. Sorry to that, i'm just newbee) Fixed.